experiments/AMIP/components/land/bucket_init.jl

```diff
@@ -159,6 +159,29 @@ function ClimaLSM.initialize_drivers(a::CoupledRadiativeFluxes{FT}, coords) wher
     return (;)
 end
 
+"""
+    temp_anomaly_aquaplanet(coord)
+
+Introduce a temperature IC anomaly for the aquaplanet case.
+The values for this case follow the initial conditions of the ClimaAtmos
```

Comment:
After Zhaoyi's confirmation, I'd say "follow the moist Help-Suarez setup of Thatcher & Jabplonowski (2016, eq. 6), consistent with ClimaAtmos aquaplanet." Otherwise LGTM.